docs/user-guide/volumes.md

```diff
@@ -487,7 +487,7 @@ spec:
   - name: test-volume
     # This VMDK volume must already exist.
     vsphereVolume:
-      volumePath: myDisk
+      volumePath: "[Datastore] myDisk"
```

Comment:
Most likely myDisk is going to be inside some folder. How about "[DatastoreName] volumes\myDisk"